test/DynamoCoreUITests/RecordedTests.cs

```diff
@@ -46,8 +46,6 @@ public override void Init()
             // to create our own copy of Controller here with command file path.
             DynamoPathManager.Instance.InitializeCore(
               Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
-
-            DynamoPathManager.PreloadAsmLibraries(DynamoPathManager.Instance);
```

Comment:
The base class `DSEvaluationViewModelUnitTest` method `StartDynamo` takes care of this.